admission-server/package.json

```diff
@@ -12,7 +12,7 @@
     "@hono/node-server": "1.11.0",
     "@wundergraph/cosmo-shared": "workspace:^",
     "dotenv": "^16.4.5",
-    "hono": "4.2.7"
+    "hono": "4.6.12"
```

Comment:
Die you test it end to end?

Reply:
@JivusAyrus ping

Reply:
Nope, was still looking at a few things. Will do the test in the end.

Reply:
Tested cdn and the admission server, it works. The rest is just vitest